src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>You are about to permanently delete this {{ term }}. After you delete it, it will no longer appear on Dockstore.</p>
+  <p>A {{ term }} can be deleted if it:</p>
+  <ul>
+    <li>Has never been published,</li>
+    <li>Was created on/after April 1, 2022, and</li>
+    <li>Is not a checked or checker workflow.</li>
+  </ul>
+
+  <p>
+    After you delete the {{ term }}, you must uninstall the Dockstore GitHub App from the source GitHub repo, or edit/remove the
+    .dockstore.yml file so that it no longer describes the {{ term }}. If you do not, the {{ term }} may reappear on Dockstore the next time
```

Comment:
The dialog should probably only display this paragraph if the entry was created via the .dockstore.yml scheme, otherwise it could be a little confusing.